Web.md

```diff
@@ -133,6 +133,107 @@ A MIME type mismatch, a response whose
 [rejected](http://tc39.github.io/ecma262/#sec-rejectpromise) with a
 `TypeError`.
 
+### Structured serialization of a `WebAssembly.Memory`
+
+A `WebAssembly.Memory` is a
+[serializable object](https://html.spec.whatwg.org/multipage/infrastructure.html#serializable-objects), and will require changes to the
+[HTML Standard](https://html.spec.whatwg.org/multipage/infrastructure.html#serialization-steps) as follows:
+
+[StructuredSerializeInternal](https://html.spec.whatwg.org/multipage/infrastructure.html#structuredserializeinternal) adds a new step after step 12:
+
+* Otherwise, if *value* has a [[WebAssembly.Memory]] internal slot, then:
+  1. Let *size* be *value*.[[BufferObject]].[[ArrayBufferByteLength]].
+  2. Let *maximum* be *value*.[[WebAssembly.Memory]].[[maximum]].
```

Comment:
IIUC, [[maximum]] is "reaching into" the memory defined by the core wasm spec.  In that case, could you add a link to https://webassembly.github.io/spec/exec/runtime.html#memory-instances, rename to `max` and note that the value is a `u32?` (optional `u32`)?